docs/core/pipes/multi-value.pipe.md

```diff
@@ -0,0 +1,30 @@
+# [Multi Value Pipe](../../../lib/core/pipes/multi-value.pipe.ts "Defined in multi-value.pipe.ts")
```

Comment:
There is a mix in the documentation between the new pipe and the metadata card documentation I guess part of it belong to the metadata. Plus this pipe needs to be add in the index of the readme